File: IT/Stringhe/Esercizi/Java/Esercizio5.java
/* file Esercizio5.java */

public class Esercizio5 {
	public static void main(String [] args) {
		StringBuilder sb = new StringBuilder("Ciao, mondo!");
		String s = sb.reverse().toString();

		for (int i = 0; i < s.length(); i++) {
			System.out.print(s.charAt(i));
		}

		System.out.print(System.lineSeparator());
	}
}

File: IT/1/Stringhe/Esercizi/C++/esercizio5.cpp
/*
file esercizio5.cpp
compilare con: g++ -std=c++11 esercizio5.cpp
*/

#include <iostream>
#include <algorithm>

using std::reverse;
using std::string;
using std::cout;
using std::endl;

int main()
{
	string s = "Ciao, mondo!";

	reverse(s.begin(), s.end());

	for (char c : s) {
		cout << c;
	}

	cout << endl;
}

